### ISSUE TITLE
Problem when ics have EXDATE and COUNT on RRULE

### ISSUE THREAD

**Issue by munkex** [u01jmg3/ics-parser#145]
Hey,

We have noticed problems with the parser (or maybe we are doing something wrong).

1) When a file contains one or more `EXDATE` in a event those dates won't be removed from the returned data, but will still be shown as events in the returned array.
example:

```
DTSTART;TZID=Europe/Copenhagen:20171009T090000
DTEND;TZID=Europe/Copenhagen:20171009T113000
RRULE:FREQ=WEEKLY;UNTIL=20171020T070000Z;BYDAY=MO,TU,WE,TH,FR
EXDATE;TZID=Europe/Copenhagen:20171016T090000
EXDATE;TZID=Europe/Copenhagen:20171017T090000
EXDATE;TZID=Europe/Copenhagen:20171018T090000
EXDATE;TZID=Europe/Copenhagen:20171019T090000
EXDATE;TZID=Europe/Copenhagen:20171020T090000
```

2) When `RRULE` is set and `COUNT` is set to 5 it will count first day event + 5 days, which should be 5 days total not 1 + 5  days.
Example: `RRULE:FREQ=WEEKLY;COUNT=5;BYDAY=MO,TU,WE,TH,FR`

Heres the file we are talking about, try and import it into google calendar or apple calendar or similar program. and then try and run it through the parser and you should be able to produce the same problem.

- http://martin.phpserver.dk/test_with_exdate.ics
- http://martin.phpserver.dk/test_with_single_exdate.ics

We are using it like this (so maybe we are doing something wrong)

```
$ical   = new ICal(false, array(
                'defaultSpan'           => 2,     // Default value
                'defaultTimeZone'       => 'UTC',
                'defaultWeekStart'      => 'MO',  // Default value
                'skipRecurrence'        => false, // Default value
                'useTimeZoneWithRRules' => false, // Default value
            ));
$test 	= $ical->initURL($calendar_url . '?' . time());

$events = $ical->eventsFromInterval('52 week');
```

**Comment by u01jmg3** [comment]
1. Was unable to reproduce this behaviour with the provided iCal. The parser correctly excludes the listed exclusion dates.
2. Fixed the issue of wrong event counts with 5f85732

**Comment by munkex** [comment]
Seems like it is working correctly now :) 👍
